docs/index.bs

```diff
@@ -2971,6 +2975,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       1. Let |preloadResponse| be a new [=promise=].
       1. Let |workerRealm| be null.
       1. Let |eventHandled| be null.
+      1. Let |crossOriginIsolatedCapability| be |request|'s [=request/client=]'s [=environment settings object/cross-origin isolated capability=].
```

Comment:
Has someone familiar with isolation looked at this? Two timings are taken, once from "in parallel", and another from the service worker thread. Neither of these are the request's client, so does the request's client's cross-origin isolated capability apply here?

Reply:
It's how things are done in FETCH. Not sure if it applies to service workers. in the same way. @annevk ?

Reply:
This is interesting as there might be a mismatch between the service worker agent and the requesting agent. I suspect we have to consult both and only if they are both true you would get the high-resolution time. Does that make sense?

That might be a change from what is implemented today as that probably only considers the requesting agent.

(Fetch uses the requesting agent exclusively as it doesn't create any agents of its own.)